megamek/src/megamek/common/weapons/ArtilleryBayWeaponIndirectFireHandler.java

```diff
@@ -86,6 +86,33 @@ public boolean cares(IGame.Phase phase) {
         }
         return false;
     }
+        
+    @Override
+    protected void useAmmo() {
+        for (int wId : weapon.getBayWeapons()) {
+            Mounted bayW = ae.getEquipment(wId);
+            // check the currently loaded ammo
+            Mounted bayWAmmo = bayW.getLinked();
+
+            if (bayWAmmo == null) {// Can't happen. w/o legal ammo, the weapon
+                // *shouldn't* fire.
+                System.out.println("Handler can't find any ammo!  Oh no!");
```

Comment:
You will want to log this instead of using System.out.println

Reply:
This is a spot of code I found and repurposed where logging hasn't been changed yet. What's the new format supposed to be?  Is it the Try {   Catch ?  I've seen a few of those and I think I could change to use that.

Reply:
There should be a member variable called "game", which probably has a logger instance. You don't need a try-catch block or anything like that. Wildj79 provided some good examples in the contributors channel just a few minutes ago: https://megamek-public.slack.com/files/U2FKHG8EL/F7UG5SZD0/log4j_example.java